docs/install.md

```diff
@@ -99,6 +99,11 @@ Once installed, issue `sudo service docker start` to start the docker service in
 
     It appears to be that next versions of WSL2 kernels will support KVM.
 
+### Mac OS
+Containerlab doesn't run on mac OS because Docker isn't natively supported.
```

Comment:
thanks @Spiritus44 
can you maybe change this line to

"... because Docker Desktop for Mac doesn't provide the networking features containerlab relies on."

Reply:
Done !